services/hanke-service/src/test/resources/fi/hel/haitaton/hanke/hakemus/hakemusUpdateRequest-blank.json

```diff
@@ -0,0 +1,14 @@
+{

```

Comment:
This could as well be a static object in Kotlin. But that way, any changes would cause compilation errors instead of errors trying to read the JSON during tests.